GameDefinition/Content/ItemKind.hs

```diff
@@ -133,7 +133,7 @@ spike2 = spike
                , SetFlag MinorEffects
                , Odds (10 * 1 `dL` 10) [] [toVelocity 70] ]
                    -- at deep levels sometimes even don't limit velocity
-  -- , idesc    = ""
+  , idesc    = "A jagged spur of metal."
```

Comment:
I don't know if it's github's fault or my fault, but we ended up with "A jagged spur of metal." and I very much prefer "A jagged skewer of rusty metal."
```suggestion
  , idesc    = "A jagged skewer of rusty metal."
```

Reply:
It will be Github's fault. It's been undoing things at me all day. I'll fix it again.